README.md

```diff
@@ -12,9 +12,9 @@ See [action.yml](action.yml) and the [Pages starter workflows][starter-workflows
 
 In order to release a new version of this Action:
 
-1. Locate the semantic version of the upcoming release (a draft is maintained by the [`draft-release` workflow][draft-release])
+1. Locate the semantic version of the [upcoming release][release-list] (a draft is maintained by the [`draft-release` workflow][draft-release])
 
-2. Push a matching tag, for instance for `v0.1.0`:
+2. Push a matching tag (e.g.`v0.1.0`) **if** different than the `main` branch (if not, publishing the draft release will create it):
```

Comment:
Really? That's cool, do we even need this whole step then?

Reply:
No, not really. I didn't do it at all for the latest release.

Reply:
I published without yesterday. Let's remove the step. Publishing a release does it for us.

Reply:
Revised. 👍🏻